### PR TITLE
Initial rerender

### PR DESCRIPTION
**Pull request by jklynch**
MNT: Re-rendered with conda-build 3.19.0, conda-smithy 3.6.14, and conda-forge-pinning 2020.03.20

https://github.com/NSLS-II/centroids

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,30 +13,30 @@ build:
...
-    - pybind11
     - gxx_linux-64
     - cpplint
+    - pybind11
 
   run:
-    - lmfit-c
-    - pandas
     - python
-    - {{ pin_compatible('numpy') }}
```
Comment (mrakitin) [inline recipe/meta.yaml:34]: I think this still needs to be here. I remember we struggled with this configuration a lot while developing this recipe in the lightsource2-recipes repo.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,30 +13,30 @@ build:
...
-    - scipy
-    - pytest
-    - pytest-runner
+    - pip
+    - setuptools
     - cmake
-    - pandas
-    - lmfit-c
-    - pybind11
     - gxx_linux-64
```
Comment (mrakitin) [inline recipe/meta.yaml]: This can also be parametrized as in the `lmfit-c` feedstock.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -13,30 +13,30 @@ build:
   script: python setup.py install
 
 requirements:
+  host:
+    - python
+    - pip
+
   build:
-    - setuptools
     - python
```
Comment (mrakitin) [inline recipe/meta.yaml:24]: Normally it should be enough to have `python` and `pip` in the host section. Why were they needed here again?

**Review by mrakitin — COMMENTED** [review]
Here are a few comments regarding recent updates in the `meta.yaml` file.

**Comment by jklynch** [comment]
I'll mark this ready for review after I am done working on it.

**Comment by mrakitin** [comment]
The failing builds are due to the randomly failing tests. Intend to merge and hope it works.

**Pull request merged**